ablog/blog.py

```diff
@@ -121,6 +121,7 @@ def verify_fn(key, value, config):
     ("disqus_shortname", None, True),
     ("disqus_drafts", False, True),
     ("disqus_pages", False, True),
+    ("blog_post_pattern", "", True, require_config_type(str)),
```

Comment:
I don't quite understand what this is for.

Reply:
check out the `conf.py` for what it's used for. basically you can put a glob-like pattern, and any docnames that match this pattern will be treated as blogposts (equivalent to `blogpost: true` in their front-matter)